### PR TITLE
Set OpenGL_GL_PREFERENCE to LEGACY

### PR DESCRIPTION
**Pull request by daleglass**
GLVND appears to break the entire UI on ATI video cards.

It was set to GLVND due to an understanding that this was a no-op setting, but it turns
out the documentation in OpenGL_GL_PREFERENCE is confusing, and it looks like LEGACY
is the setting that was being used after all.

An additional point of interest is that debian/rules contained OpenGL_GL_PREFERENCE=GLVND,
and this was done back in the High Fidelity days. It's uncertain whether this was correct
or not, though.

### CODE REVIEW

**Comment by JulianGro** [comment]
I tested this and it indeed fixed my problems with QML in AppImage builds on master.

File: interface/resources/qml/controls/+webengine/FlickableWebViewCore.qml [hunk 1]
```diff
@@ -16,7 +16,6 @@ Item {
     property alias webViewCoreProfile: webViewCore.profile
     property string webViewCoreUserAgent
 
-    property bool useBackground: webViewCore.useBackground
```
Comment (ctrlaltdavid) [inline interface/resources/qml/controls/+webengine/FlickableWebViewCore.qml:19]: Is this line intentionally removed?

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-fe53ad2_8963aefd.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-fe53ad2.tar_8963aefd.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-fe53ad2.tar_8963aefd.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-fe53ad2_8963aefd.exe

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-fe53ad2_8963aefd.dmg

File: interface/resources/qml/controls/+webengine/FlickableWebViewCore.qml (hunk 1 above)
Comment (kasenvr) [reply interface/resources/qml/controls/+webengine/FlickableWebViewCore.qml:19]: Was it perhaps unnecessary?

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-f15d22a_7b6fbe1f.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-f15d22a.tar_7b6fbe1f.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-f15d22a.tar_7b6fbe1f.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-f15d22a_7b6fbe1f.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR806/Vircadia-Alpha-PR806-f15d22a_7b6fbe1f.exe

**Comment by JulianGro** [comment]
@kasenvr I don't think we are going to get any more QA than me testing out the release builds. If there is any issue to find we will likely only find it after a release when lots of different distributions and hardware configurations run this.

**Comment by kasenvr** [comment]
I'll at least give it a run on Windows. :) Gimme a moment.

**Comment by JulianGro** [comment]
Yeah I guess we should just talk about this in the dev meeting

**Review by kasenvr — APPROVED** [review]
Build seems to run fine for me.

**Pull request merged**